### ISSUE TITLE
Segfault on OS X after upgrading from 0.1.13 to 0.1.15

### ISSUE THREAD

**Issue by JayKickliter** [commercialhaskell/intero#123]
In the process of trying to get intero to work again I completely blew away any trace of haskell and emacs on my system and started from scratch, so I don't have much more information than the title. After starting from scratch, `stack exec intero` would still `segfault 11`. I was able to get it working again by once again blowing away all traces of stack/haskell/emacs from the system and transferring them from a laptop that I haven't touched in a couple weeks.

**Comment by chrisdone** [comment]
So is there an issue with Intero?

**Comment by JayKickliter** [comment]
I thought so, but It would be the first time I'm wrong. I'll try upgrading to the latest intero again and see if the same thing happens. 

Sorry for my poorly worded issue. It makes no sense to me reading it now. If it wasn't clear, intero worked perfectly for me right out of the box when I installed it a week or two ago. It starting segfaulting when I upgraded to intero 0.1.15 without (knowingly) changing anything else about my system.
